src/Symfony/Bundle/FrameworkBundle/Command/BuildDebugContainerTrait.php

```diff
@@ -38,8 +38,10 @@ protected function getContainerBuilder(KernelInterface $kernel): ContainerBuilde
         if ($this->containerBuilder) {
             return $this->containerBuilder;
         }
+        $container = $kernel->getContainer();
+        $containerDump = $container->hasParameter('debug.container.dump') ? $container->getParameter('debug.container.dump') : null;
 
-        if (!$kernel->isDebug() || !(new ConfigCache($kernel->getContainer()->getParameter('debug.container.dump'), true))->isFresh()) {
+        if (null === $containerDump || !(new ConfigCache($containerDump, true))->isFresh()) {
```

Comment:
should we pass `$kernel->isDebug()` as the second argument of the config cache ?

Reply:
indeed, updated